### DISCUSSION TITLE
Can't get --toggle zoom-fullscreen or native-fullscreen working with skhd

### DISCUSSION THREAD

**Discussion by edgrgomes** [asmvik/yabai#2069] [Q&A]
Hello,

As the title says I can't get the toggles for fullscreen working with skhd.
If i run them in a terminal they work but not with shortcut, I tried various shortcuts
Im in sonoma 14.2.1
Terminal I use kitty, zsh 

PS: Can't also get yabai --start-service and --stop-service working through skhd

Thank you for the help!
![Imagem WhatsApp 2024-01-29 às 23 21 10_6fd4988a](https://github.com/koekeishiya/yabai/assets/100764115/990d8403-3a11-48f8-8928-28dde88a07b8)
![Imagem WhatsApp 2024-01-29 às 23 21 10_b420a7f0](https://github.com/koekeishiya/yabai/assets/100764115/092116b7-8263-4917-900d-4fba0a9f4d87)

**Comment by frankyxhl** [comment]
I figured out the following as an alternative way.

`yabai -m window --grid 1:1:0:0:1:1`

**Reply by Bellavene** [reply]
This works only for unmanaged windows
And better to use this `yabai -m window --grid 1:1:0:0:1:1 || yabai -m window --toggle zoom-fullscreen`
So it will maximize an unmanaged window or toggle it for managed ones.

**Reply by asmvik** [reply]
Pretty sure you can do `yabai -m window --toggle windowed-fullscreen` that works on both managed and unmanaged windows, in newer yabai versions.

**Reply by Bellavene** [reply]
@koekeishiya Works great, thank you!
